tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/KonanToolRunner.kt

```diff
@@ -100,5 +105,8 @@ internal class KonanInteropRunner(project: Project)
     }
 }
 
-internal class KonanCompilerRunner(project: Project) : KonanCliRunner("konanc", "Kotlin/Native compiler", project)
-internal class KonanKlibRunner(project: Project) : KonanCliRunner("klib", "Klib management tool", project)
+internal class KonanCompilerRunner(project: Project, additionalJvmArgs: List<String> = emptyList())
+    : KonanCliRunner("konanc", "Kotlin/Native compiler", project, additionalJvmArgs)
+
+internal class KonanKlibRunner(project: Project, additionalJvmArgs: List<String> = emptyList())
+    : KonanCliRunner("klib", "Klib management tool", project, additionalJvmArgs)
```

Comment:
Maybe named params look nicer here, what do you think? (same for the `KonanCompilerRunner` of course)

Reply:
I'm not sure. Kotlin doesn't allow using names only for part of arguments so I cannot write something like
```
(toolName = "klib", fullName ="Klib management tool", project, additionalJvmArgs)
```
All the arguments must be passed with names so we get something like this:
```
( ... project = project, additionalJvmArgs = additionalJvmArgs)
```

Also IntelliJ IDEA can show hints for parameter names. It does so for this code too and the first two args look better when the file is opened in the IDE.